_episodes/03-high-dim-modeling.md

```diff
@@ -52,16 +130,44 @@ FIXME
 > {: .solution}
 {: .challenge}
 
-# Filter Methods
+In the above challenge, we saw that we could use a priori knowledge about our dataset to remove features that provide no information to the model. However, what if we don't know which features will be relevant? We could test out each possible subset of features by iteratively selecting different feature subsets prior to training and testing our models. However, if there are d features, brute force testing would require training and evaluating (2^d - 1) different models! In the next section, we will learn about a few different approaches to feature selection that allow us to avoid this brute force method.
```

Comment:
For better version control I'd suggest breaking up the lines more.  Example below.
```suggestion
In the above challenge, we saw that we could use a priori knowledge 
about our dataset to remove features that provide no information to 
the model. However, what if we don't know which features will be relevant? 
We could test out each possible subset of features by iteratively selecting 
different feature subsets prior to training and testing our models. 
However, if there are d features, brute force testing would require training 
and evaluating (2^d - 1) different models! In the next section, we will learn 
about a few different approaches to feature selection that allow us to avoid
 this brute force method.
```